Source/PlasticSourceControl/Private/PlasticSourceControlProjectSettings.h

```diff
@@ -20,4 +20,8 @@ class UPlasticSourceControlProjectSettings : public UDeveloperSettings
 	/** Hide the domain part of an username e-mail address (eg @gmail.com) if the UserNameToDisplayName map didn't match (enabled by default). */
 	UPROPERTY(config, EditAnywhere, Category = "Plastic SCM")
 	bool bHideEmailDomainInUsername = true;
+
+	/** If enabled, you'll be prompted to check out changed files */
```

Comment:
Add something like "(enabled by default)" at the end of the comment, since that is what the user will see as a tooltip